exporters/pelorus/__init__.py

```diff
@@ -28,8 +28,12 @@ def _print_version():
     """
     import __main__
 
-    # name of dir above app.py
-    exporter_name = pathlib.PurePath(__main__.__file__).parent.name
+    file = getattr(__main__, "__file__", None)
+    if file:
+        # name of dir above app.py
+        exporter_name = pathlib.PurePath(file).parent.name
+    else:
+        exporter_name = "INTERPRETER"
```

Comment:
This was necessary to get `import pelorus` working in a REPL. I can split this into a separate PR if wanted.